libs/langchain-openai/src/chat_models.ts

```diff
@@ -435,8 +437,14 @@ export class ChatOpenAI<
       fields?.configuration?.organization ??
       getEnvironmentVariable("OPENAI_ORGANIZATION");
 
+<<<<<<< Updated upstream
```

Comment:
You should remove these git artifacts before merging: `<<<<<<<`, `=======`, `>>>>>>> Stashed changes`

Reply:
My bad, forgot to check this after rebase. Fixed